spec/features/topics_spec.rb

```diff
@@ -28,9 +28,29 @@
     click_link("New Topic")
     fill_in 'Title', with: 'animals'
     click_button 'Create Topic'
-    expect(page).to render(:topic)
+    expect(page).to have_text "New topic successfully created!"
     expect(page).to have_css("h4", text: "animals")
   end
+
+  scenario "user can edit a topic" do 
+    topic = create(:topic, title: "animals", user: @user)
+    visit topics_path
+    click_link("animals")
+    click_link("Edit Topic")
+    fill_in "topic_title", with: "wild animals" 
```

Comment:
Add expectation to test that the topic_title text field contains the old value "animals" before changing it to "wild animals".